doc/classes/VisualScriptSelf.xml

```diff
@@ -1,8 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualScriptSelf" inherits="VisualScriptNode" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Outputs a reference to the current instance.
 	</brief_description>
 	<description>
+		This node provides a reference to the node running the script.
```

Comment:
`Provides a reference to the node running the script.` / running the visual script
I'd add an example: does it mean in visual script, you need this to access the current node's methods and parameters? Is it just the equivalent of `self`? I'm not sure what this does.